changes/v5.0.16/fix-9832.zh.md

```diff
@@ -0,0 +1 @@
+[FIXME] failure to handle timeout error in resource worker.
```

Comment:
You can use https://www.deepl.com/translator to make a first attempt at translating. So far I have not got any complainant for the translations that it has generated for me.